src/ontology/mondo-edit.obo

```diff
@@ -544908,6 +544908,14 @@ intersection_of: MONDO:0020066 ! Ehlers-Danlos syndrome
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2198 ! COL1A2
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8760" xsd:anyURI
 
+[Term]
+id: MONDO:0100613
+name: BMP4-related ocular growth disorder
+def: "Any ocular growth disorder in which the cause of the disease is a mutation in the BMP4 gene." [https://clinicalgenome.org/affiliation/40077/]
```

Comment:
Something is missing. 
if "ocular growth disorder" is in the name and/or the definition, we need to have "ocular growth disorder" defined. 
We don't have a term for this, and it is not explained in the current definition. 

This might require input from the requester.

Reply:
@sabrinatoro I added a new class 'ocular growth disorder' (the requester provided a definition) and reclassified several terms as children (per my google research)